### PR TITLE
Update google-closure-compiler

### PR DESCRIPTION
**Pull request by sbc100**
This change was generated by running:

 `npm install google-closure-compiler@latest`

I the removed the `^` which `npm install` put that beginning of the
version so that we are precise in the version we are using.

### CODE REVIEW

**Comment by sbc100** [comment]
Needed for https://github.com/emscripten-core/emscripten/pull/12343

File: package.json [hunk 1]
```diff
@@ -5,10 +5,10 @@
     "ws": "~0.4.28"
   },
   "dependencies": {
-    "google-closure-compiler": "20200224.0.0",
+    "acorn": "7.3.1",
+    "google-closure-compiler": "20200920.0.0",
```
Comment (RReverser) [inline package.json:9]: Oh hah, I didn't realise it's that simple, thought you're using a separate JAR file somewhere. Thanks!

**Comment by sbc100** [comment]
CI is green so I imagine this should be good to land?

**Review by RReverser — APPROVED** [review]
Nice PR number btw! 😀

**Pull request merged**